### PR TITLE
Minor style fixes & rename

### PR DESCRIPTION
**Pull request by Geod24**
Better reviewed commit-by-commits. One commit is a bit more meaty (the rename), otherwise it's pretty trivial. Contain typos spotted by @AndrejMitrovic in #815 .

### CODE REVIEW

File: source/agora/consensus/EnrollmentManager.d [hunk 1]
```diff
@@ -22,17 +22,17 @@
     In order to ensure that we will never loose the ability to reveal
     pre-images, this implementation uses a reproducible scheme:
     on the first run, we generate a "cycle seed" which is derived from a hash
-    of the private key, a constant string, and a nounce, starting from 0.
+    of the private key, a constant string, and a nonce, starting from 0.
```
Comment (AndrejMitrovic) [inline source/agora/consensus/EnrollmentManager.d:25]: In the UK a `nonce` is also an insult. For example "You're such a nonce" means "you're so stupid".

_The More You Know...!_

File: source/agora/test/Ledger.d [hunk 2]
```diff
@@ -132,46 +132,37 @@ unittest
     // transactions are ordered lexicographically by hash in the Merkle tree
     hashes.sort!("a < b");
 
-    const Hash ha = hashes[0];
```
Comment (MichaelKim20) [inline source/agora/test/Ledger.d:135]: I imagined a tree like this.
```
               abcdefgh
        +---------++---------+
      abcd                  efgh
   +---++---+            +---++---+
  ab        cd          ef        gh
+-++-+    +-++-+      +-++-+    +-++-+  
a    b    c    d      e    f    g    h
```

Comment (AndrejMitrovic) [reply source/agora/test/Ledger.d:135]: I think the mapping between characters [a .. z] is simple to [0 .. $]. He only replaced the bottom level.

**Review by AndrejMitrovic — APPROVED** [review]

File: source/agora/test/Ledger.d (hunk 2 above)
Comment (MichaelKim20) [reply source/agora/test/Ledger.d:135]: Yeah, I got it.

**Pull request merged**